Support/CMake/Toolchain-WinStore-x86.cmake

```diff
@@ -0,0 +1,17 @@
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+set(CMAKE_SYSTEM_NAME WindowsStore)
+set(CMAKE_SYSTEM_VERSION 10.0)
```

Comment:
You might wanna set all the variables explicitly in all toolchain files instead of relying on implicit value. That's just a personal preference thing though, so feel free to ignore.